.ddev/commands/web/orchestrate.d/20_cleanup_default_plugins.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+wp plugin is-installed akismet && wp plugin uninstall akismet
+wp plugin is-installed hello && wp plugin uninstall hello
```

Comment:
We could go one step further and also check the activation status of the plugins.
```suggestion
wp plugin is-active akismet && wp plugin uninstall akismet --deactivate
wp plugin is-active hello && wp plugin uninstall hello --deactivate
```

Not that this would be much useful. 😆

Reply:
I actually do want to destructively get rid of this bloat